content/05.prior-knowledge.md

```diff
@@ -1,79 +1,82 @@
+Supervised neural networks used in medical imaging studies [@doi:10.1016/j.procs.2016.07.014] (reviewed in [@doi:10.1098/rsif.2017.0387]), which are trained to predict labels or classes, are also an example of representation learning.
 Learned features in the medical imaging domain may be a series of edges representing a blood vessel formation that discriminates between disease states.
 Features learned from transcriptomic data could be coordinated sets of genes involved in a biological process that are descriptive in some way [@doi:10.1038/s41467-020-14666-6].
 
-In the rare disease domain, Dincer et al. leveraged publicly available acute myeloid leukemia (AML) gene expression data to improve the prediction of _in vitro_ drug responses [@doi:10.1101/278739]. 
-The authors trained a variational autoencoder (an unsupervised neural network that learns a series of representations from data), or VAE, on AML data that had been collected over time without the desired phenotypic information (drug response). 
+In the rare disease domain, Dincer et al. leveraged publicly available acute myeloid leukemia (AML) gene expression data to improve the prediction of _in vitro_ drug responses [@doi:10.1101/278739].
+The authors trained a variational autoencoder (an unsupervised neural network that learns a series of representations from data), or VAE, on AML data that had been collected over time without the desired phenotypic information (drug response).
 The authors used the learned attributes to encode a low-dimensional representation of held-out AML data with phenotype labels of interest, and used this representation as input to a classifier that predicted _in vitro_ drug response.
 
 Representation learning tends to be data-intensive; many samples are required.
 Though there were over 6500 AML samples from many different studies used as part of the training set in Dincer et al. [@doi:10.1101/278739], we expect that in other rare diseases considerably fewer samples will be available or may be from different tissues in systemic diseases.
 The study by Dincer and colleagues highlights another challenge: samples collected as part of multiple studies may not be associated with the deep phenotypic information that would maximize their scientific value.
 In the next section, we will introduce methods or approaches that may be more broadly useful in rare diseases; representation learning underlies many of them.
 
+<!-- TODO: Is this header level right? I can't tell if this is supposed to be a subsection or its own section. The strategies discussed here might either be considered bringing both statistical + prior knowledge & data together or another part of using prior knowledge + data. I think I favor the former, though it would require putting a brief transfer and multi-task blurb in the prior/related data section. -->
```

Comment:
@cgreene if we make my suggested change to the header level, you are envisioning that a brief TL and MTL blurb goes into this markdown file (e.g., a paragraph gets inserted at line 2 that includes this blurb) - did I follow this comment correctly?

Reply:
I think then we should have a brief explanation of what transfer learning is and a brief explanation of what multi-task learning is. Then we would have a separate final section where we highlight Dincer et al. and Taroni et al.. That provides the opportunity to talk about regularization + other data + other knowledge all together, bridging the statistical section (first part) with the knowledge and data section (second part).

Reply:
Tracked in #110 and #111.